### DISCUSSION TITLE
DoubleTime + Difficulty Adjust UI bug?

### DISCUSSION THREAD

**Discussion by 0x84f** [ppy/osu#17532] [Q&A] [answered]
If you select DoubleTime you don't get an option to adjust the rate of it, if you apply the difficulty adjusting mod the option appears.
However when you remove the difficulty adjusting mod (but you leave DoubleTime on) the rate changing option still remains.
[Video of issue](https://streamable.com/cp1qu1)

**Comment by bdach** [comment, marked as answer]
it's not a bug. the customization options show on their own if a mod requires changing a setting to do anything useful.

click the customization button on the bottom of the mod overlay to customize DT alone. this will hopefully be conveyed better in an upcoming redesign of this screen.

**Reply by 0x84f** [reply]
Thanks, I thought it was weird that it was like that!
